### PR TITLE
Casr report: "Stdin" field

### PR DESCRIPTION
**Pull request by Avgor46**

### CODE REVIEW

File: src/bin/casr-san.rs [hunk 1]
```diff
@@ -134,6 +134,10 @@ fn main() -> Result<()> {
     report.executable_path = argv[0].to_string();
     report.proc_cmdline = argv.join(" ");
     let _ = report.add_os_info();
+    if let Some(mut file_path) = stdin_file.clone() {
```
Comment (SweetVishnya) [inline src/bin/casr-san.rs:137]: We should do the similar in casr-gdb.

**Review by SweetVishnya — CHANGES_REQUESTED** [review]

File: src/bin/casr-san.rs (hunk 1 above)
Comment (Avgor46) [reply src/bin/casr-san.rs:137]: Done

**Comment by SweetVishnya** [comment]
We should add some tests for new stdin field.

**Comment by Avgor46** [comment]
> We should add some tests for new stdin field.

Added 2 tests for stdin field (with casr-san report and casr-gdb report)

**Pull request merged**